File: py/company/hr/recruitment/interview.py
# interview module
def schedule_interview(candidate_name):
    print(f'''INFO: Scheduling interview for "{candidate_name}"''')

def conduct_interview(candidate_name):
    print(f'''INFO: Conducting interview for "{candidate_name}"''')

def feedback_interview(candidate_name, feedback):
    print(f'''INFO: Feedback for "{candidate_name}": {feedback}''')

# Example usage if this module is run directly
if __name__ == "__main__":
    candidate = "John Doe"
    schedule_interview(candidate)
    conduct_interview(candidate)
    feedback_interview(candidate, "Excellent")

File: go/company/hr/recruitment/interview/interview.go
package interview

import (
	"fmt"
	"strconv"
)

func ScheduleInterview(candidateName string) {
	fmt.Printf("Scheduling interview for %s\n", strconv.Quote(candidateName))
}

func ConductInterview(candidateName string) {
	fmt.Printf("Conducting interview for %s\n", strconv.Quote(candidateName))
}

func FeedbackInterview(candidateName, feedback string) {
	fmt.Printf("Feedback for %s: %s\n", strconv.Quote(candidateName), feedback)
}
